requests/__init__.py

```diff
@@ -56,7 +56,7 @@
 # See https://github.com/kennethreitz/requests/issues/3213 as well.
 import ssl
 
-if ssl.OPENSSL_VERSION_INFO < (1, 0, 1):
+if hasattr(ssl, 'OPENSSL_VERSION_INFO') and getattr(ssl, 'OPENSSL_VERSION_INFO') < (1, 0, 1):
```

Comment:
No worries @drpoggi. I think we're all striving to be better at this F/OSS collaboration thing.